app/services/food_service.rb

```diff
@@ -0,0 +1,10 @@
+class FoodService
+  def self.search_by_ingredient(ingredient)
+    response = Faraday.get('https://api.nal.usda.gov/fdc/v1/foods/search') do |req|
+      req.params['api_key'] = ENV['api_key']
+      req.params['query'] = ingredient
+      req.params['pageSize'] = 10
+    end
```

Comment:
Great job setting these query params in the block here. Nice and readable/maintainable.